tests/e2e/multicard/test_offline_inference_distributed.py

```diff
@@ -190,3 +190,19 @@ def test_models_distributed_Qwen3_W8A8():
             quantization="ascend",
     ) as vllm_model:
         vllm_model.generate_greedy(example_prompts, max_tokens)
+
+
+def test_models_distributed_Qwen3_W4A8DYNAMIC():
+    example_prompts = [
+        "Hello, my name is",
+    ]
+    max_tokens = 5
+
+    with VllmRunner(
+            snapshot_download("vllm-ascend/Qwen3-8B-W4A8"),
+            max_model_len=8192,
+            dtype="auto",
+            tensor_parallel_size=4,
```

Comment:
sorry, we have changed the CI resource, tp only work with tp=2 now, can you rebase to main and update this value?